impeller/aiks/aiks_unittests.cc

```diff
@@ -1289,6 +1289,28 @@ TEST_P(AiksTest, CanDrawAnOpenPath) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+TEST_P(AiksTest, CanDrawAnOpenPathThatIsntARect) {
```

Comment:
Is this one of the playground tests that also makes a golden?

Reply:
IIRC these are all playground tests, so we should see a golden delta pop up in a bit.